src/components/features/reviews-section/action/review.action.ts

```
@@ -0,0 +1,24 @@
+"use server";
+
+import { ReviewInput } from "@lib/schemas/review.schema";
+import { getAccessToken } from "@lib/utils/get-token.util";
+import { log } from "console";
```

ESLint

[error] 'log' is defined but never used.

(@typescript-eslint/no-unused-vars)